### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by mayoreee**

### CODE REVIEW

File: docs/full-node/manage-a-jmes-validator/faq.mdx [hunk 1]
```diff
@@ -96,8 +96,9 @@ Once a validator is created and registered, bJMES holders can delegate bJMES to
   - This Keypair is generated when a node is created with `jmesd init`.
   - The Private Key can be found in the `priv_validator_key.json` file in the `config` directory after running `jmesd init`.
   - The Public Key is derived from the Private Key and can be found and seen by running the command `jmesd tendermint show-validator`
-
-    **Example:** `jmesvalconspub1zcjduc3qcyj09qc03elte23zwshdx92jm6ce88fgc90rtqhjx8v0608qh5ssp0w94c`
```
Comment (Alex-Werner) [inline docs/full-node/manage-a-jmes-validator/faq.mdx:100]: Let's keep it no ? 
Its good people know what a validator example looks like

File: docs/full-node/run-a-full-jmes-node/system-config.mdx [hunk 2]
```diff
@@ -63,33 +64,12 @@ As the network grows, the minimum hardware requirements will also grow. It is re
...
 </p>
 </details>
 
-- Linux users: `sudo apt-get install -y build-essential`
-
-## Commonly used ports
-
-jmesd uses the following TCP ports. Toggle their settings to match your environment.
-
-Most validators will only need to open the following port:
```
Comment (Alex-Werner) [inline docs/full-node/run-a-full-jmes-node/system-config.mdx:79]: Why do we remove this ?

Comment (mayoreee) [reply docs/full-node/run-a-full-jmes-node/system-config.mdx:79]: Confusion. :)

Too many information so far has been confusing per feedback.

The intention now is to keep the guide as a simple, step-by-step guide to getting a validator running - even for non-technical readers.

We could add technicalities (not required for starting a basic validator) like this on a separate "Advanced Setup" guide for more technical folks - that should come later.

File: docs/full-node/manage-a-jmes-validator/faq.mdx (hunk 1 above)
Comment (mayoreee) [reply docs/full-node/manage-a-jmes-validator/faq.mdx:100]: I replaced this with an actual JMES validator address. Not removed.

Screenshot 2023-05-09 at 21 49 16

Comment (Alex-Werner) [reply docs/full-node/manage-a-jmes-validator/faq.mdx:100]: The screen has an issue. 
You example the validator one below a show-address. (It seems to be mixed-up). 
show-validator will have `jmesval....`, show-address will just have `jmes...`

Comment (mayoreee) [reply docs/full-node/manage-a-jmes-validator/faq.mdx:100]: `show-address` returns bech_32 address (consensus address)
`show-validator` return s validator info
Screenshot 2023-05-14 at 22 41 03

Example:
Screenshot 2023-05-14 at 22 41 38
Screenshot 2023-05-14 at 22 42 12

Comment (mayoreee) [reply docs/full-node/manage-a-jmes-validator/faq.mdx:100]: @Alex-Werner

**Pull request merged**